micrometer-core/src/main/java/io/micrometer/core/instrument/util/package-info.java

```diff
@@ -14,8 +14,8 @@
  * limitations under the License.
  */
 @NonNullApi
-@NonNullFields
+@NullMarked
```

Comment:
I haven't done the migration for `@NonNullApi` yet and can't do it automatically until the code compiles. It doesn't currently due to the NullAway errors.